easybuild/easyblocks/c/clang.py

```diff
@@ -237,6 +242,15 @@ def configure_step(self):
         else:
             self.cfg['configopts'] += "-DLLVM_ENABLE_ASSERTIONS=OFF "
 
+        if self.cfg["build_lld"]:
+            self.cfg['configopts'] += "-DLLVM_ENABLE_PROJECTS=lld "
```

Comment:
@ocaisa I don't think this works, does it?

Should be

```python
self.cfg.update('configopts', "-DLLVM_ENABLE_PROJECTS=lld")

Reply:
It's the same as existing code, see 3 lines up. Should I change it everywhere?